geode-assembly/src/acceptanceTest/java/org/apache/geode/client/sni/ClientSNICQAcceptanceTest.java

```diff
@@ -107,9 +111,18 @@ public void before() throws IOException, InterruptedException {
 
   }
 
+  @After
+  public void after() throws Exception {
+    String output =
+        docker.exec(options("-T"), "geode", arguments("cat", "server-dolores/server-dolores.log"));
+    System.out.println("Server log file--------------------------------\n" + output);
```

Comment:
do we need this log in the test output?

Reply:
Dan thought it was a good idea for the SingleServer test & I thought so, too.  If something goes wrong you probably want this output in the logs.